client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/FamilyMembers/FamilyMembersTable/FamilyMembersTable.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody, Checkbox, Typography } from '@material-ui/core';
+
+import InvolvedContact from 'models/InvolvedContact';
+import useFamilyContactsUtils from 'Utils/FamilyContactsUtils/useFamilyContactsUtils';
+
+import useStyles from './FamilyMembersTableStyles';
+import { FamilyContactsTableHeaders } from '../../../../../FamilyContactsDialog/FamilyContactsTable/FamilyContactsTableHeaders';
+
+const FamilyMembersTable: React.FC<Props> = (props: Props) => {
+    
+    const { familyMembers } = props;
+    const classes = useStyles();
+
+    const { convertToIndexedRow, getTableCell } = useFamilyContactsUtils();
+
+    const [selectedFamilyMembers, setSelectedFamilyMembers] = useState<InvolvedContact[]>([]);
+
+    const FamilyTableHeadersWithCheckbox = [''].concat(Object.values(FamilyContactsTableHeaders));
+
+    useEffect(() => {
+        familyMembers.map((familyMember: InvolvedContact) => {
+            familyMember.selected = false;
+        });
+    }, []);
+
```

Comment:
change to familyMembers.map((familyMember: InvolvedContact) => ({
            ...familyMember,
            selected = false;
        }));